modules/simple_forms_api/app/services/simple_forms_api/pdf_uploader.rb

```diff
@@ -17,6 +17,9 @@ def upload_to_benefits_intake(params)
       uuid_and_location = get_upload_location_and_uuid(lighthouse_service, form)
       create_form_submission_attempt(params, uuid_and_location)
 
+      # Validate document before uploading
```

Comment:
This method might be dead code?
https://github.com/department-of-veterans-affairs/vets-api/pull/18363/files#diff-090bd1db19a34784e8717b327c19ba59e4d7fb6b37b8f797a43e80bd68965bcdL130